cheetah/particles.py

```diff
@@ -1018,6 +1018,140 @@ def from_twiss(
             device=device,
             dtype=dtype,
         )
+    
+    @classmethod
+    def uniform_3d_ellispoid(
```

Comment:
This function has since been added on `master` and actually merged onto this branch as well. So this copy here should be removed and only the one from `master` should remain.